src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -179,6 +182,19 @@ public static Vector4 Overlay(Vector4 backdrop, Vector4 source)
         return Vector4.Min(Vector4.One, new Vector4(cr, cg, cb, 0));
     }
 
+    /// <summary>
+    /// Returns the result of the "Overlay" compositing equation.
+    /// </summary>
+    /// <param name="backdrop">The backdrop vector.</param>
+    /// <param name="source">The source vector.</param>
+    /// <returns>The <see cref="Vector4"/>.</returns>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static Vector256<float> Overlay(Vector256<float> backdrop, Vector256<float> source)
+    {
+        Vector256<float> color = OverlayValueFunction(backdrop, source);
+        return Avx.Min(Vector256One, Avx.Blend(color, Vector256<float>.Zero, BlendAlphaControl));
```

Comment:
We're "missing" a JIT optimization here so swapping the parameter order for `vblendps` can be more efficient

Namely do `Avx.Blend(Vector256<float>.Zero, color, 0b_01_11_01_11);`. This is special to `Zero` and `AllBitsSet` since they aren't "normal" constants but are instead always generated into register. For other constants the original ordering you had would've been better.

Reply:
I'll log a JIT issue to track ensuring we support commutativity for `Blend`

Reply:
Awesome. Thanks!

Reply:
https://github.com/dotnet/runtime/issues/82365